package.json

```diff
@@ -70,7 +69,7 @@
     "jest-snapshot-serializer-ansi": "^1.0.0",
     "jsonlint": "^1.6.3",
     "prettier": "1.18.2",
-    "tmp": "0.1.0"
+    "uuid": "^3.3.3"
```

Comment:
Consider [nanonid](https://github.com/ai/nanoid) for size and performance.

Reply:
Will check it out.